src/UI/Component/Symbol/Factory.php

```diff
@@ -89,21 +87,21 @@ public function icon(): Icon\Factory;
 	 *   usage:
 	 *       1: Glyphs MUST NOT be used in content titles.
 	 *       2: >
-	 *          Glyphs MUST be used for cross-sectional functionality such as mail for
+	 *          Glyphs MUST be used for cross-sectional functionality such as Mail for
```

Comment:
Why is mail capitalised here? We only capitalise names of other UI Components in the description and rules, to make them stand out.